lib/dirac.js

```diff
@@ -471,6 +471,10 @@ dirac.tx = {
       tx[ tbl ] = new Dal( client );
       tx[ tbl ].befores = dirac.dals[ tbl ].befores;
       tx[ tbl ].afters  = dirac.dals[ tbl ].afters;
+      tx[ tbl ].lock = function(mode, callback) {
```

Comment:
Every transaction is recreating new DALs scoped to the same client. Maybe we can make it so that every atomic action has a way to override client

like 

```
dirac.dals.users.find({}, { _client: client}, callback )
```